containers/Dockerfile.EESSI-build-node-debian-sid

```diff
@@ -0,0 +1,48 @@
+ARG cvmfsversion=2.11.2
+ARG archspecversion=0.2.2
+ARG awscliversion=1.32.22
+ARG fuseoverlayfsversion=1.10
+
+FROM debian:sid-20240330-slim AS prepare-deb
+ARG cvmfsversion
+COPY ./containers/build-or-download-cvmfs-debs.sh /build-or-download-cvmfs-debs.sh
+RUN sh /build-or-download-cvmfs-debs.sh ${cvmfsversion}
+
+
+FROM debian:sid-20240330-slim
+ARG cvmfsversion
+ARG archspecversion
+ARG awscliversion
+ARG fuseoverlayfsversion
+
+COPY --from=prepare-deb /root/deb /root/deb
+
+RUN apt-get update
+RUN apt-get install -y sudo vim openssh-client gawk autofs curl attr uuid fuse3 libfuse2 psmisc gdb uuid-dev lsof strace
+# python3 and jq are required for eessi-upload-to-staging script (next to awscli)
+RUN apt-get install -y python3-pip python3-venv jq
+RUN dpkg -i /root/deb/cvmfs_${cvmfsversion}~1+debian13_$(dpkg --print-architecture).deb \
+            /root/deb/cvmfs-fuse3_${cvmfsversion}~1+debian13_$(dpkg --print-architecture).deb \
+            /root/deb/cvmfs-libs_${cvmfsversion}~1+debian13_$(dpkg --print-architecture).deb \
+            /root/deb/cvmfs-config-default_latest_all.deb \
+            /root/deb/cvmfs-config-eessi_latest_all.deb
+
+# download binary for specific version of fuse-overlayfs
+RUN curl -L -o /usr/local/bin/fuse-overlayfs https://github.com/containers/fuse-overlayfs/releases/download/v${fuseoverlayfsversion}/fuse-overlayfs-$(uname -m) \
+  && chmod +x /usr/local/bin/fuse-overlayfs
+
+RUN echo 'CVMFS_QUOTA_LIMIT=10000' > /etc/cvmfs/default.local \
+  && echo 'CVMFS_CLIENT_PROFILE="single"' >> /etc/cvmfs/default.local \
+  && echo 'CVMFS_HIDE_MAGIC_XATTRS=yes' >> /etc/cvmfs/default.local
+
+RUN mkdir -p /cvmfs/software.eessi.io
+
+RUN useradd -ms /bin/bash eessi
+
+RUN python3 -m venv /opt/archspec && . /opt/archspec/bin/activate && pip3 install archspec==${archspecversion} && deactivate && ln -s /opt/archspec/bin/archspec /usr/local/bin/archspec
+# stick to awscli v1.x, 2.x is not available through PyPI (see https://github.com/aws/aws-cli/issues/4947)
+RUN python3 -m venv /opt/awscli && . /opt/awscli/bin/activate && pip3 install awscli==${awscliversion} && deactivate && ln -s /opt/awscli/bin/aws /usr/local/bin/aws
+
+RUN curl -OL https://raw.githubusercontent.com/EESSI/eessi-bot-software-layer/develop/scripts/eessi-upload-to-staging \
+  && mv eessi-upload-to-staging /usr/bin \
+  && chmod a+x /usr/bin/eessi-upload-to-staging
```

Comment:
There's probably a good reason to have `eessi-upload-to-staging` in the build container (maybe it was already in existing containers). The bot has a config setting for this (`artefact_upload_script`). If we would change that script in the `EESSI/eessi-bot-software-layer` repository, we may need to rebuild the container? Alternatively, one could download the script at the start of the container?

It's probably no big issue if the script is a bit outdated in the container? Debugging issues may be a bit tricky, e.g., if one assumes the version from the `EESSI/eessi-bot-software-layer` is used, but for some reason a different version installed in the container is used.

Reply:
It's indeed here for legacy reasons, as we found it useful in the past to have that script available. Perhaps it could still be useful for RISC-V (while we don't have a bot yet and try some things manually), but I'm also fine with removing it.

Reply:
Well, we can easily download it to the RISC-V machine if necessary, so I've removed the script from both build containers in https://github.com/EESSI/filesystem-layer/pull/132/commits/8da6758a033e8dc44d781a3b83bfad25a20470b5.